apps/wallet/src/ui/app/shared/forms/controls/PasswordInput.tsx

```diff
@@ -24,6 +24,7 @@ export const PasswordInput = forwardRef<HTMLInputElement, PasswordInputProps>(
 					ref={forwardedRef}
 				/>
 				<ButtonOrLink
+					tabIndex={-1}
```

Comment:
this was added so the Tab key skips over the show / hide icon. not 100% sure if there's a better way to handle this so that it's still accessible

Reply:
so you can easily tab between enter / confirm password fields